instance_config.yaml

```diff
@@ -39,6 +39,18 @@ veda_s3_hotstore_bucket:
 veda_deliverable_bucket:
 # Settings
 multi_upload_barrier: 2000000000
+veda_base_url:
+s3_base_url: https://s3.amazonaws.com
+
+
+# transcript bucket config
+transcript_bucket_access_key:
+transcript_bucket_secret_key:
+transcript_bucket_name:
```

Comment:
We don't need the access key or secret keys any more -- this should be handled with the terraform provisions for roles (in this case, 'veda_deliver')